react-components/src/tools/manage/ManageTools.js

```diff
@@ -425,9 +440,10 @@ function ToolListing(props) {
                         >
                             <TableCell padding="none">
                                 <IconButton
-                                    onClick={() =>
-                                        presenter.onShowToolInfo(tool.id)
-                                    }
+                                    onClick={(event) => {
+                                        event.stopPropagation();
```

Comment:
This one tricky. Without this, every second click on the  same`info` will fail. See `onToolSelection` where selection is set to `null` when clicked the second time.

Reply:
Just found another bug with this. Working on a fix....

Reply:
I went back and removed the logic that toggles selection on click. Since we don't have multi selection checkbox, I don't think we need it.

Reply:
This is ready for review now.